pylon-test/src/main/java/io/github/pylonmc/pylon/test/GenericTest.java

```diff
@@ -0,0 +1,11 @@
+package io.github.pylonmc.pylon.test;
+
+import org.bukkit.NamespacedKey;
+
+public interface GenericTest {
+    default NamespacedKey getKey() {
+        return new NamespacedKey(TestAddon.instance(), this.getClass().getName());
```

Comment:
```suggestion
        return new NamespacedKey(TestAddon.instance(), this.getClass().getName().toLowercase(Locale.ROOT));
```
Don't forget the import